packages/paste-core/components/input-box/__tests__/input-chevron-wrapper.test.tsx

```diff
@@ -8,13 +8,13 @@ expect.extend(matchers);
 
 describe('HTML attributes', () => {
   it('should set a element data attribute for InputChevronWrapper', () => {
-    render(<InputChevronWrapper element="INPUT">input-chevron</InputChevronWrapper>);
-    expect(screen.getByText('input-chevron').getAttribute('data-paste-element')).toEqual('INPUT_CHEVRON_WRAPPER');
+    render(<InputChevronWrapper element="INPUT_ICON_WRAPPER">input-chevron</InputChevronWrapper>);
+    expect(screen.getByText('input-chevron').getAttribute('data-paste-element')).toEqual('INPUT_ICON_WRAPPER');
```

Comment:
I would like to keep the element name as `x_CHEVRON_WRAPPER` in these type of inputs. Mainly because it's not just the input icon, it's specifically the input chevron, the select chevron, the combobox chevron, etc

Reply:
Okey, that's fair. I will update 😸 I'm assuming the same for `Select` then too @SiTaggart?

Reply:
Yup